fastestimator/trace/io/traceability.py

```diff
@@ -438,14 +437,23 @@ def _document_models(self) -> None:
     def _document_sys_config(self) -> None:
         """Add a system config summary to the traceability document.
         """
+        env_dict = get_environment()
```

Comment:
this can all be reverted since not used in new trace

Reply:
fixed